lib/server.ml

```diff
@@ -184,19 +184,23 @@ let error t handle code =
        t.channel.write t.reply
     )
 
-let serve t (type t) block (b:t) =
```

Comment:
Sounds good, I'll do that. We discussed that it's fine to set RO to true by default, because writing never really worked until recently (https://github.com/xapi-project/nbd/pull/46)